SteakBot.Core/Services/MemeService.cs

```diff
@@ -52,6 +66,37 @@ public bool EditCommand(string oldCommandName, MemeCommand newCommand)
             return true;
         }
 
+        public MemeResultType GetMemeType(string value)
```

Comment:
This somehow moved above `RemoveMeme`. Please keep the method order.

Reply:
Corrected.